test/integ/test_put_get_user_stage.py

```diff
@@ -39,6 +40,72 @@ def test_put_get_small_data_via_user_stage(is_public_test, tmpdir, conn_cnx, fro
     )
 
 
+@pytest.mark.skipolddriver
+@pytest.mark.aws
+@pytest.mark.parametrize(
+    "from_path",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "accelerate_config",
+    [True, False],
+)
+def test_put_get_accelerate_user_stage(
+    is_public_test, tmpdir, conn_cnx, from_path, accelerate_config
+):
+    from snowflake.connector.file_transfer_agent import SnowflakeFileTransferAgent
+    from snowflake.connector.s3_storage_client import SnowflakeS3RestClient
+
+    """[s3] Puts and Gets Small Data via User Stage."""
```

Comment:
This has to be the first thing in the function to be considered a doc-string, right now it's just a string constant that nothing points to